src/handlers.rs

```diff
@@ -575,14 +555,11 @@ pub fn get_messages(
         limit = 256;
     }
     // Query the database
-    let raw_query: String;
+    let raw_query: &str;
     if query_params.get("from_server_id").is_some() {
```

Comment:
Not sure if you are aware, but if statements are also expressions in Rust, so you could write:
```rust
let raw_query = if query_params.get("from_server_id").is_some() {
   "SELECT id, public_key, timestamp, data, signature FROM messages WHERE id > (?1) AND is_deleted = 0 ORDER BY id ASC LIMIT (?2)"
} else {
    "SELECT id, public_key, timestamp, data, signature FROM messages WHERE is_deleted = 0 ORDER BY id DESC LIMIT (?2)"
}
```
(if you prefer)

Reply:
Thanks, appreciate any tips like this as I'm a rust noob.